plugin/asyncrun.vim

```diff
@@ -130,6 +130,13 @@ let g:asyncrun_stop = get(g:, 'asyncrun_stop', '')
 " specify how to run your command
 let g:asyncrun_mode = get(g:, 'asyncrun_mode', 0)
 
+" Use quickfix-ID to allow concurrent use of quickfix list and not
+" interleave streamed output of a running command with output from
+" other plugins
+if !exists('g:asyncrun_qf_id_available')
```

Comment:
Exposed as a configuration variable in case a user desires to switch this behavior off.

Reply:
I prefer a short one, like `g:asyncrun_qf_id` or `g:asyncrun_qfid`

Reply:
Done